### DISCUSSION TITLE
Welcome to Print discussions

### DISCUSSION THREAD

**Discussion by PeterWone** [PDConSec/vsc-print#217] [Announcements]
We're taking advantage of GitHub's new support for discussions. This is great news for everyone. It lets us properly organise how-to and feature requests with "issues" just being bug reports. 

Better yet, Q&A means you can help each other without someone deciding your question is "not a good fit". The rules are simple: 

* Try to help each other.
* If you think an opinion or question is stupid then ignore it. 
* If you _must_ comment on an answer or opinion then don't be a jerk about it, stick to the facts and remember that people are not obliged to agree with you even if you're right.
* If you think an _answer_ is wrong then write a better one. 

Please don't make us step in, we don't have time for it.

**Comment by GregO2020** [comment]
Is it possible with Print extension for Visual Studio Code to save an HTML file, when editing a ipynb Python notebook file?

**Comment by PeterWone** [comment]
Our of the box Print has document renderers for source code, Markdown and plain text. It also supports the installation of third-party document renderers.

None of our contributors are familiar with Jupyter notebooks and none of the sixteen million people who installed support for Jupyter notebooks have contributed a document renderer.

This may be your time to shine. Given the size of your potential user base it could look very good on a resume.
